test/functional/api/vim_spec.lua

```diff
@@ -1512,7 +1512,7 @@ describe('API', function()
   it("does not leak memory on incorrect argument types", function()
     local status, err = pcall(nvim, 'set_current_dir',{'not', 'a', 'dir'})
     eq(false, status)
-    ok(err:match(': Wrong type for argument 1, expecting String') ~= nil)
+    ok(err:match(': Wrong type for argument 1 when calling set_current_dir, expecting String') ~= nil)
```

Comment:
here it is `nvim_set_current_dir`

Reply:
Thanks! that fixed it, only two more to go. But these two look completely unrelated to this PR to me